File: CSharp/Array/Reverse.cs
using System;
using System.Collections.Generic;

public class Test
{
	public static void Main()
	{
            // 配列の宣言
            string[] datas = {"沖田", "武蔵", "モードレッド"};
        
            // 配列の要素順序を反転
            Array.Reverse(datas);
        
	    // 配列の中身を表示
	    System.Console.WriteLine(datas[0]); // モードレッド
	    System.Console.WriteLine(datas[1]); // 武蔵
	    System.Console.WriteLine(datas[2]); // 沖田
	}
}

File: Java/Array/Reverse.java
import java.util.*;
import java.lang.*;
import java.io.*;
import java.util.List;
import java.util.ArrayList;

class Test
{
    public static void main (String[] args)
    {       
        // 配列の宣言・初期化
        String[] name = {"吹雪", "白雪", "深雪"};
        
        // 逆順に並び替え
        List list = Arrays.asList(name);
        Collections.reverse(list);
        String[] name2 = (String[])list.toArray(new String[3]);
            
        // コンソール出力
        for(String n : name2) System.out.println(n); // 深雪, 白雪, 吹雪 
    }
}
